src/lib/remark-plugins/remark-plugin-adjust-link-urls/__tests__/remark-plugin-adjust-link-urls.test.ts

```diff
@@ -53,15 +53,19 @@ describe('remarkPluginAdjustLinkUrls', () => {
 
 		describe('pre-adjusts folder-relative urls starting with `./`', () => {
 			const testCases = [
+				// Example of test case:
+				// https://github.com/hashicorp/waypoint/blob/63d5149a2aa63214f6f5fa41e825cb1a13783b9e/website/content/plugins/aws-ecs.mdx?plain=1#L17
 				{
-					input: './configuration',
-					expected: '/docs/disks/configuration',
-					currentPath: '/docs/disks',
+					input: './pack',
+					expected: '/plugins/pack',
```

Comment:
```suggestion
					expected: '/plugins/aws-ecs/pack',
```
Am I looking at this wrong?

Reply:
nvm I saw your comment in slack

Reply:
`/plugins/aws-ecs/pack` is not a valid path to an MDX file in the repo, check out the "Example of test case" link I added in the comment above this object.